coalib/__init__.py

```diff
@@ -6,6 +6,12 @@
 
 
```

Comment:
commit shortlog: __init.py__: Add help description

im pretty sure its __init__.py, not __init.py__ :D